pkg/migrations/op_create_table.go

```diff
@@ -363,7 +364,11 @@ func (w *ConstraintSQLWriter) WriteCheck(check string, noInherit bool) string {
 	if w.Name != "" {
 		constraint = fmt.Sprintf("CONSTRAINT %s ", pq.QuoteIdentifier(w.Name))
 	}
-	constraint += fmt.Sprintf("CHECK (%s)", check)
+	if !strings.HasPrefix(check, "CHECK (") {
```

Comment:
I added this step so we can use the constraint writer in the duplication process as well.

Reply:
The `ConstraintWriter` has been moved into a separate file by an already-merged PR right (#665)?  

Maybe this PR needs to be rebased onto `main`?

Reply:
Do the tests that were added in #665 need to be updated to test this new check?